backend/package.json

```diff
@@ -11,18 +11,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@firebase/testing": "^0.20.11",
```

Comment:
According to [the npm page](https://www.npmjs.com/package/@firebase/testing), this package is deprecated